src/app-logic/l10n.js

```diff
@@ -10,28 +10,38 @@ import {
   PSEUDO_STRATEGIES_DIRECTION,
 } from 'firefox-profiler/utils/l10n-pseudo';
 
-// This contains the locales we support in the production. Don't forget to update
-// the array RTL_LOCALES when adding a RTL locale, if necessary.
-// AVAILABLE_STAGING_LOCALES is replaced, using webpack's DefinePlugin, by all
-// available locales, when running with L10N=1, using the "l10n" versions of the
-// yarn scripts. Especially this is done when building and deploying the l10n
-// branch in netlify.
-export const AVAILABLE_LOCALES: Array<string> = AVAILABLE_STAGING_LOCALES || [
-  'de',
-  'el',
-  'en-GB',
-  'en-US',
-  'es-CL',
-  'fr',
-  'ia',
-  'it',
-  'nl',
-  'pt-BR',
-  'sv-SE',
-  'uk',
-  'zh-CN',
-  'zh-TW',
-];
+// This object contains the locales that are complete at at least 90% and
```

Comment:
There are a couple of typos: double "at", and probably you mean "that **we** enable by default"?

Reply:
this double "at" was on purpose:

`Complete at -- at least -- 90%`

but is it correct english? :-)

Reply:
I don't think so. But "locales that are at least 90% complete" would work around the doubt.